test/acceptance/features/bindAppToMultipleServices.feature

```diff
@@ -38,20 +58,79 @@ Feature: Bind a single application to multiple services
+        And Secret "binding-request-1" contains "BACKEND_HOST_INTERNAL_DB" key with value "internal.db.stable.example.com"
+        And Secret "binding-request-2" contains "BACKEND_HOST_EXTERNAL_DB" key with value "external.db.stable.example.com"
+        And "myapp-2sbrs" deployment must contain SBR name "binding-request-1"
+        And "myapp-2sbrs" deployment must contain SBR name "binding-request-2"
+        And The application env var "BACKEND_HOST_INTERNAL_DB" has value "internal.db.stable.example.com"
+        And The application env var "BACKEND_HOST_EXTERNAL_DB" has value "external.db.stable.example.com"
+
+        Scenario: Bind two backend services by creating 1 SBR to a single application
+        Given Generic test application "myapp-1sbr" is running
+        * The Custom Resource is present
+        """
+        apiVersion: stable.example.com/v1
+        kind: Backend
+        metadata:
+            name: internal-db-1sbr
+            annotations:
+                servicebindingoperator.redhat.io/spec.host_internal_db: 'binding:env:attribute'
+        spec:
+            host_internal_db: internal.db.stable.example.com
+        """
+        * The Custom Resource is present
+        """
+        apiVersion: stable.example.com/v1
+        kind: Backend
+        metadata:
+            name: external-db-1sbr
+            annotations:
+                servicebindingoperator.redhat.io/spec.host_external_db: 'binding:env:attribute'
+        spec:
+            host_external_db: external.db.stable.example.com
+        """
+        When Service Binding is applied
+            """
+            apiVersion: operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: binding-request-1sbr
+            spec:
+                application:
+                    name: myapp-1sbr
+                    group: apps
+                    version: v1
+                    resource: deployments
+                services:
+                -   group: stable.example.com
+                    version: v1
+                    kind: Backend
+                    name: internal-db-1sbr
+                -   group: stable.example.com
+                    version: v1
+                    kind: Backend
+                    name: external-db-1sbr
+            """
+
+        Then jq ".status.conditions[] | select(.type=="CollectionReady").status" of Service Binding "binding-request-1sbr" should be changed to "True"
+        And jq ".status.conditions[] | select(.type=="InjectionReady").status" of Service Binding "binding-request-1sbr" should be changed to "True"
+        And Secret "binding-request-1sbr" contains "BACKEND_HOST_INTERNAL_DB" key with value "internal.db.stable.example.com"
+        And Secret "binding-request-1sbr" contains "BACKEND_HOST_EXTERNAL_DB" key with value "external.db.stable.example.com"
+        And "myapp-1sbr" deployment must contain SBR name "binding-request-1sbr"
```

Comment:
No need to check this, because the last steps check that implicitly. Also, injecting env vars via secrets is an implementation details, and user must not be aware of that. What she/he really cares is about what env vars are visible to the application.

Reply:
Done.